_data/roadmap.yml

```diff
@@ -14,19 +14,22 @@ in-progress:
   - Language server convert to/from `use` code action
   - Local Hex API key encryption
   - Unaligned bit array support on JavaScript
+  - Language server extract variable code action
+  - Language server inline variable code action
```

Comment:
I don't think this one has been started

Reply:
Oh sorry, it's the expand function capture one that has. I'll fix that